app/assets/javascripts/components/PipelineSampleReads.jsx

```diff
@@ -26,9 +26,9 @@ class PipelineSampleReads extends React.Component {
   }
 
   pipelineInProgress(status) {
-    if (status === null || this.state.rerun === true ) {
+    if (status === null || this.state.rerun === true || status === 'RUNNABLE' || status === 'RUNNING') {
       return true;
-    } else if ( status === 'ERROR' || status === 'CHECKED' || status === 'FAILED') {
```

Comment:
you should keep the checked

Reply:
Yea, i intend to add it back when I discuss where to fix the button on the reports page with Rebecca. @yunfangjuan